blog/2025/pcsx2-2.6/index.mdx

```diff
@@ -0,0 +1,606 @@
+---
+draft: false
+title: PCSX2 2.6.0 is now out!
+tags:
+  - devblog
+date: 2026-01-4
+description: PCSX2 2.6 Release Post
+hide_table_of_contents: false
+titleImage: /img/2-0-banner.webp
+image: /img/2-0-banner.webp
+authors:
+  - kamfretoz
+  - lightningterror
+  - airgamer
+  - pandubz
+  - jordan
+  - link
+  - govanify
+---
+
+Nearly 6 months after PCSX2 2.4.0 release, we are excited to unveil PCSX2 2.6.0! We're even more excited to showcase the things that we have been working on.
+
+<!--truncate-->
+
+## <TextGradient startColor="#2A6DFF" endColor="#245DD9">New Year - New Build</TextGradient>
+
+Now that the dust has settled down from [our biggest single release to date](../../2024/pcsx2-2-release/index.mdx), we decided to step back a bit and give more love and attention to the interface, Quality of Life features and Accessibility. There are also more performance improvements to be seen, which we are very excited to show you! Our contributors have been hard at work further refining the foundation that has been laid out since 2.0.0 to make PCSX2 more accessible to everyone!
```

Comment:
Should "Quality of Life" or "Accessibility" be capitalized here? They aren't proper nouns or labels in the UI that are being matched for demonstration purposes.